### PR TITLE
Working on 030340 and 030350.

### PR DESCRIPTION
**Pull request by mikerenfro**

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -433,6 +433,9 @@ rhel7stig_login_defaults:
   umask: '077'
   create_home: 'yes'
 
+# RHEL-07-030350
+rhel7stig_auditd_mail_acct: sa-isso-list@site.mil
```
Comment (jamescassell) [inline defaults/main.yml]: probably best to leave the default as 'root'

**Comment by shepdelacreme** [comment]
Did you rebase this on devel or did you merge devel into your branch?

**Comment by mikerenfro** [comment]
I probably rebased. Will get the conflicts resolved regardless.

**Comment by shepdelacreme** [comment]
Yeah git operations can be confusing. I still have to spend a lot of time googling to get things right.

I think you did a pull instead of a rebase on this one since all these commits that are showing up are ones merged into the devel tree by other PRs. And it has your commits in this branch set prior to the already merged commits.

I was going to suggest backing out these commits but at this point it is probably not worth the effort. I will do a squash and merge to keep the tree clean.

If you want a good doc on rebasing I've found this one to be useful.
https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request

File: defaults/main.yml [hunk 2]
```diff
@@ -436,9 +436,15 @@ rhel7stig_login_defaults:
 # RHEL-07-030330: set this to 25% of the free space in /var/log/audit (measured in megabytes)
 rhel7stig_auditd_space_left: 225
 
+# RHEL-07-030350
+rhel7stig_auditd_mail_acct: root
+
 # RHEL-07-020630
 rhel7stig_homedir_mode: 0750
 
+# RHEL-07-030350
```
Comment (shepdelacreme) [inline defaults/main.yml]: Can you remove line 445 to 447? It seems to be duplicated from line 439 to 441.

**Pull request merged**